Base/QTCore/qSlicerAbstractCoreModule.h

```diff
@@ -204,7 +204,7 @@ class Q_SLICER_BASE_QTCORE_EXPORT qSlicerAbstractCoreModule : public QObject
   virtual void printAdditionalInfo();
 
   /// Convenience method to return slicer wiki URL
-  QString slicerWikiUrl()const{ return "http://www.slicer.org/slicerWiki/index.php"; }
+  QString slicerWikiUrl()const{ return "https://www.slicer.org/slicerWiki/index.php"; }
```

Comment:
My comment in https://github.com/Slicer/Slicer/pull/6276/commits/4934ca63e8f4e8f1b548771d17266f1d3896ee08#r844439362 was motivated by the fact trying to browse to `https://www.slicer.org/slicerWiki/index.php` fails.

In this form, the following works `https://www.slicer.org/slicerWiki/index.php/`

That said https://www.slicer.org/w/index.php works as expected and do not return a `404`